client/my-sites/earn/home.tsx

```diff
@@ -151,7 +154,8 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		return {
 			title,
 			body,
-			badge: translate( 'New' ),
+			badge:
+				moment( userDate ).diff( moment( '2019-11-26 00:00:00' ) ) < 0 ? translate( 'New' ) : null,
```

Comment:
Could we use [`.isAfter` or `.isBefore`](https://momentjs.com/docs/#/query/is-before/) here to make it clearer what this is checking? Also, you don't need to wrap the second bit in `moment`, moment will automatically convert the string to a moment internally.